Makefile

```diff
@@ -68,10 +68,14 @@ $(OUT)/trivialDAO_storage.tz : admin_address = tz1QozfhaUW4wLnohDo6yiBUmh7cPCSXE
 $(OUT)/trivialDAO_storage.tz : token_address = tz1QozfhaUW4wLnohDo6yiBUmh7cPCSXE9Af
 $(OUT)/trivialDAO_storage.tz : now_val = Tezos.now
 $(OUT)/trivialDAO_storage.tz : metadata_map = (Big_map.empty : metadata_map)
+$(OUT)/trivialDAO_storage.tz : fixed_proposal_fee_in_token = 0n
+$(OUT)/trivialDAO_storage.tz : ledger = (Big_map.empty : ledger)
```

Comment:
We should calculate the correct `total_supply` for a given ledger.

In order to be able to do so we need to `fold` over the parameter here, but that's not possible to do with a `big_map`, so my suggestion is to take a `list` instead for this option and then in the Ligo code `List.fold` over it to create the actual `ledger` `Big_map` and the `total_supply` as well.